packages/orbit-components/src/Switch/index.tsx

```diff
@@ -19,7 +19,7 @@ const StyledSwitchBase = styled.div<{ checked?: boolean; disabled?: boolean }>`
     cursor: ${!disabled && "pointer"};
     width: 40px;
     height: 20px;
-    background-color: ${theme.orbit.paletteCloudNormal};
+    background-color: ${theme.orbit.paletteCloudDark};
```

Comment:
maybe it's good to actually create token for this inside tokens

Reply:
Agreed. Will do